### PR TITLE
Focus on conformance tests for dind

### PR DESCRIPTION
**Pull request by Q-Lee**
Conformance tests are far more reliable. Let's focus on those for e2e tests atm.

### CODE REVIEW

File: kubetest/dind/dind.go [hunk 1]
```diff
@@ -127,7 +127,8 @@ func (d *DindDeployer) NewTester() (*Tester, error) {
 // Test just execs ginkgo. This will take more parameters in the future.
 func (t *Tester) Test() error {
 	skipRegex := "--skip=\"(Feature)|(NFS)|(StatefulSet)\""
-	args := []string{"--seed=1436380640", "--nodes=10", skipRegex, t.e2etest,
+	focusRegex := "--focus=\".*Conformance.*\""
```
Comment (BenTheElder) [inline kubetest/dind/dind.go]: the usual flag is `\[Conformance\]`?

Comment (Q-Lee) [reply kubetest/dind/dind.go]: Canonically yes.

**Pull request merged**